tests/cypress.json

```diff
@@ -16,16 +16,16 @@
   "experimentalInteractiveRunEvents": true,
   "testFiles": [
     "run-before.spec.js",
-    "create-functions.spec.js",
+    "test-prepare-functions.spec.js",
```

Comment:
Maybe we want to leave it as `prepare-functions` or `run-prepare-functions` to indicate that this is not a regular test but preparation suite